todo_app/app.py

```diff
@@ -3,21 +3,87 @@
 from todo_app.flask_config import Config
 
 import todo_app.data.session_items as session_items
+import requests, os, json
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+class Item:
+    def __init__(self,title,id,status):
+        self.title = title
+        self.status = status
+        self.id = id
+
+    def create_card(title, list_category):
+        list_id = get_list_id(list_category)
+        payload = {'key':os.environ.get('TRELLO_KEY'),'token':os.environ.get('TRELLO_TOKEN'), \
+        'idList':list_id}
+        url = api_prefix + '1/cards'
+        card_title = {"name": title}
+        api_response = requests.post(url, params=payload, data = card_title)
+        api_response_list = json.loads(json.dumps(api_response.json()))
+        return Item(api_response_list["name"],api_response_list["id"], list_category)
+     
+api_prefix = 'https://api.trello.com/'
+
+def call_api(api_suffix):
+    payload = {'fields':'name','key':os.environ.get('TRELLO_KEY'),'token':os.environ.get('TRELLO_TOKEN')}
+    url = api_prefix + api_suffix
+    api_response = requests.get(url, params=payload)
+    api_response_list = json.loads(json.dumps(api_response.json()))
+    return api_response_list
+
+def get_board_id():
+    api_suffix_get_board = '1/members/vaibhavsharma206/boards'
```

Comment:
We definitely want to avoid hardcoding your username in here - switching that for `e/members/me/boards` would be the easiest solution here.

It's worth thinking about switching this method to getting the board id from an environment variable though - it could be confusing that the board used is always the 'first' board that Trello returns, so we may find ourselves having to delete boards to get this program working.

Reply:
I didn't think of that. I have added two env variables now - User Name (program can be run for any user), and Board Name (to fetch cards from any board for that user).